### ISSUE TITLE
Updating to new grid spec

### ISSUE THREAD

**Issue by constantinpape** [mobie/centrioles-tomo-datasets#1]
@martinschorb I can update to the new grid spec. Which branch should I update?

**Comment by martinschorb** [comment]
I am waiting for https://github.com/mobie/mobie-viewer-fiji/issues/512 to be resolved.
I then want to play around with a few grids with 5x5 or 7x7 meaningful centrioles (the longest/widest/... ones for each entity).
If I use your current python tools, I assume they produce grids according to the new spec already?
The ones we have in `grid-test-tomo` were merely to play around. If you want to update them, go ahead. I am curious about the performance gain...
I will then  use another branch for the smaller grids. I am curious about the performance gain...

**Comment by constantinpape** [comment]
> If I use your current python tools, I assume they produce grids according to the new spec already?

Yes, if you use the newest commit the grids will be in the correct format.
(And remember to clear `~/.mobie/*.json` beforehand so that the jsonschema files that are used for validation are up-to-date as well).

**Comment by constantinpape** [comment]
Hey @martinschorb,
updating to the new spec here is pending the grid spec update. Probably no need to rush this, but once we have all the new spec stuff merged and MoBIE-beta up and running we should revisit this.
